frontend/package-lock.json

```diff
@@ -10,9 +10,18 @@
         "@testing-library/jest-dom": "^5.17.0",
         "@testing-library/react": "^13.4.0",
         "@testing-library/user-event": "^13.5.0",
+        "axios": "^1.4.0",
+        "bootstrap": "^5.3.1",
```

Comment:
how about we use react-bootstrap instead of bootstrap.. 
you can go through react-bootstrap for documentations